packages/tupaia-web/src/components/Modal.tsx

```diff
@@ -34,12 +37,13 @@ const CloseButton = styled(IconButton)`
   min-width: initial;
   position: absolute;
   top: 0.1rem;
-  right: 0.1rem;
+  right: 0rem;
 `;
 
 const Paper = styled(MuiPaper)`
-  background-color: ${({ theme }) => theme.palette.background.default};
-  padding: 0;
+  background-color: #202124;
+  padding-left: 3.125rem;
+  padding-right: 3.125rem;
```

Comment:
If we make these changes on here on the Modal component then they will apply everywhere that uses the modal component which includes things like the LoginModal, RegisterModal etc. I think that is correct for the background color but not for the padding. But the good news is you can just make the padding change in the ProjectsModal component instead.